src/utils.ts

```diff
@@ -242,6 +262,69 @@ export const parseTx = (tx: RawTx): FullTx => {
   return parsedTx;
 };
 
+/**
+ * Syncs the latest mempool
+ *
+ * @generator
+ * @yields {MempoolEvent}
+ */
+export async function* syncLatestMempool(): AsyncGenerator<MempoolEvent> {
+
+  logger.info(`Downloading mempool...`);
+  let success = true;
+  const mempoolResp = await downloadMempool();
+
+  if (!mempoolResp.success) {
+    yield {
+      success,
+      type: 'wait',
+      message: 'Could not download from mempool api',
+    };
+    return;
+  }
+
+  for (let i = 0; i < mempoolResp.transactions.length; i++) {
+    const tx = await downloadTxFromId(mempoolResp.transactions[i]);
+
+    if (tx === null) {
+      yield {
+        type: 'wait',
+        success: false,
+        message: `Failure on transaction ${mempoolResp.transactions[i]} in mempool`,
+      };
+      return;
+    }
+
+    const preparedTx: PreparedTx = prepareTx(tx);
+
+    try {
+      const sendTxResponse: ApiResponse = await sendTx(preparedTx);
+      if (!sendTxResponse.success) {
+        logger.debug(sendTxResponse);
```

Comment:
Maybe use `logger.error` instead of `logger.debug`?

Reply:
Done.